docs/DEVELOPER_GUIDE.md

```diff
@@ -1 +1,179 @@
-todo
\ No newline at end of file
+# Development
```

Comment:
For all of the markdown files, how about use `doctoc` to generate a toc for the document?

Reply:
get the doctoc working looks nice :), we will need to update other docs, going to send a separate PR.